docs/getting-started/user-guides/using-decrediton.md

```diff
@@ -129,4 +129,4 @@ The settings tab is also where you come to change your private passphrase.
 
 The help tab contains various links to get help using Decrediton. The "Issues" link will take you the to the Decrediton Github page where problems with the Decrediton software can be reported. We would be grateful for your help!
 
-![Help Tab](/img/decrediton/help.jpg)
\ No newline at end of file
+![Help Tab](/img/decrediton/help.jpg)
```

Comment:
Can you revert whatever this line change is, otherwise looks good.

Reply:
Will do -- seems like github's markdown editor did something to the EOF/EOL character.